src/components/BrowseInventoryFilters/BrowseInventoryFilters.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { useContext } from 'react';
+import { flowRight, omit } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes-core';
+
+import withFacets from '../../withFacets';
+import { DataContext } from '../../contexts';
+import { buildManifestObject } from '../../routes/buildManifestObject';
+import { parseFiltersToStr } from '../../utils';
+import { InstanceFiltersBrowse } from '../InstanceFilters';
+
+const BrowseInventoryFilters = ({
+  activeFilters = {},
+  applyFilters,
+  fetchFacets,
+  resources,
+  searchIndex,
+}) => {
+  const data = useContext(DataContext);
+
+  const filtersData = {
+    ...data,
+    browseType: searchIndex,
+    onFetchFacets: fetchFacets(data),
+    parentResources: resources,
+    query: {
+      query: activeFilters.query,
+      filters: parseFiltersToStr(omit(activeFilters, ['qindex', 'query'])),
+    }
+  };
+
+  return (
+    <>
+      <InstanceFiltersBrowse
+        activeFilters={activeFilters}
+        data={filtersData}
+        onChange={({ name, values }) => applyFilters(name, values)}
+        onClear={(name) => applyFilters(name, [])}
+      />
+    </>
+  );
+};
+
+BrowseInventoryFilters.manifest = Object.freeze({
+  query: buildManifestObject().query,
```

Comment:
do we really need stripes-connect here?

Reply:
Yes, to display filters facets we should provide `query` resource for existing <InstanceFiltersBrowse> component